### DISCUSSION TITLE
Selection Mode not working?

### DISCUSSION THREAD

**Discussion by rgouette** [Krypton-Suite/Standard-Toolkit#1779] [General]
Hi, not sure if this control suite is still accepting questions, but here goes:
I cannot get the selection mode to only allow ONE selection, when using the KryptonCheckedListBox. (C# desktop app)
I've set it in the properties, and also tried programmatically with: 
`            lb1.SelectionMode = Krypton.Toolkit.CheckedSelectionMode.One; in my Form load
`

**Comment by giduac** [comment]
Hi @rgouette,

Thanks for posting.
The KCheckedListBox does not support the multiselect feature.

Please create a Feature Request.

Thank you.

**Reply by rgouette** [reply]
Hi, thanks for the reply.
Just to be clear, this functionality is NOT available?
I was looking to only be able to select ONE box..
![image](https://github.com/user-attachments/assets/e4cc994b-af15-468e-b3da-ba2e9ba75ff5)

**Comment by giduac** [comment]
you mean lie in a radio group / only check one ?

**Reply by rgouette** [reply]
yeah, limit the KCheckedListBox to 1 selection..
~R

**Comment by giduac** [comment]
No, but you can code that yourself by using the

```cs
        private void kryptonCheckedListBox1_ItemCheck(object sender, ItemCheckEventArgs e)
        {

        }
```

**Comment by giduac** [comment]
Hi @rgouette,

Did that work for you?

**Reply by rgouette** [reply]
Hi, sorry for the delayed response..
I tried setting  "CheckedSelectionMode" there, but so far no luck..
I'm not a guru , so maybe missing something obvious..
~R

**Comment by giduac** [comment]
Hi @rgouette,

There's is no options in the control that does what you are looking for.
But with the event mentioned earlier you can code that kind of behaviour yourself.
Unfortunately  we don't do those kind of requests....

**Reply by rgouette** [reply]
I'm assuming development/support stopped for it..
![image](https://github.com/user-attachments/assets/011e2bba-25fc-48fb-bf64-8179f49d030c)


oh well, thanks anyway
~R

**Comment by PWagner1** [comment]
@rgouette 

Will open a new feature request for this. It won't be in the upcoming V90, but possibly V100 or V110.

cc. @giduac & @Smurf-IV
